hack/build-dev-env.sh

```diff
@@ -13,6 +13,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+set -eu
```

Comment:
this is failing `./hack/verify-boilerplate.sh` script, please add a line before this